src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -15,19 +18,47 @@
             "Phone numbers can only contain numbers, and should be at least 3 digits long";
     public static final String PHONE_VALIDATION_REGEX = "\\d{3,}";
     public final String value;
+    public final ArrayList<String> phonelist;
 
     /**
      * Validates given phone number.
      *
      * @throws IllegalValueException if given phone string is invalid.
      */
+
+    public Phone(List<String> phone) throws IllegalValueException {
+        requireNonNull(phone);
+        String phoneStr = "";
+        ArrayList<String> phones = new ArrayList<String>();
```

Comment:
As with above, the change will look like this:
`List<String> phones = new ArrayList<>();`